Makefile

```diff
@@ -71,9 +72,11 @@ wc:
 	find src tests -name '*.hs' | xargs wc
 
 content:
-	curl -XPOST http://localhost:8080/api/manage-state/create-init
-	curl -XPOST http://localhost:8080/api/manage-state/create-demo
-	curl -XPOST http://localhost:8080/api/manage-state/create-votes
+	curl -XPOST $(AULA_URL)/api/manage-state/create-init
+	curl -c cookie-jar -F /login.user=admin -F /login.pass=pssst $(AULA_URL)/login
+	curl -b cookie-jar -XPOST $(AULA_URL)/api/manage-state/create-demo
+	curl -b cookie-jar -XPOST $(AULA_URL)/api/manage-state/create-votes
+	rm cookie-jar
```

Comment:
awesome!  i've wanted to figure out how to do this for a long time.  thank you!  (-: